### ISSUE TITLE
Issue with training FCN_Vgg16_32s

### ISSUE THREAD

**Issue by interactivetech** [aurora95/Keras-FCN#60]
Hi there.

Nice job on the repo! I am trying to train the FCN_Vgg16_32s model, and I am having issue with the loss being "nan". 

I followed the steps to set up the VOC dataset and transfer VGG weights for the FCN model.

The only code changes I made was in train.py was setting model_name='FCN_Vgg16_32s' and the loss after a few steps, results to nan.

Here is a screenshot of my terminal output.

<img width="809" alt="screen shot 2018-01-09 at 10 20 09 pm" src="https://user-images.githubusercontent.com/5833510/34754385-6514fddc-f58b-11e7-8f13-924a6e882285.png">

Please advise on how I should go about debugging the loss function, and if you need additional information.

**Comment by interactivetech** [comment]
I studied the loss during training, and from 4-9 steps, the loss increased to a large number. I had a feeling it was a bad learning rate, so I investigated what was the learning rate other repos set to train FCN_VGG16-32s. It seems that people use `1e-10`.

Now the learning rate is behaving appropriately. 
Here are some links for additional details:

Pytorch Repo: https://github.com/wkentaro/pytorch-fcn/blob/cfad9e594e2bb5a327ba838a7103abeb74e190c8/torchfcn/ext/fcn.berkeleyvision.org/siftflow-fcn32s/solver.prototxt#L10

Pytorch Repo: https://github.com/ZijunDeng/pytorch-semantic-segmentation/blob/4a1721f9a3284788336430efb140288096c6dd09/train/voc-fcn/train.py#L27
